extensions/objects/templates/math-expression-content-editor.directive.ts

```diff
@@ -59,13 +59,17 @@ angular.module('oppia').directive('mathExpressionContentEditor', [
           // Naturally MathJax works asynchronously, but we can add processes
           // which we want to happen synchronously into the MathJax Hub Queue.
           MathJax.Hub.Queue(['Typeset', MathJax.Hub, outputElement[0]]);
+          ctrl.numberOfElementsInQueue++;
           MathJax.Hub.Queue(function() {
             if (outputElement[0].getElementsByTagName('svg')[0] !== undefined) {
               ctrl.svgString = (
                 outputElement[0].getElementsByTagName('svg')[0].outerHTML);
             }
-            ctrl.value.mathExpressionSvgIsBeingProcessed = false;
-            $scope.$apply();
+            ctrl.numberOfElementsInQueue--;
+            if (ctrl.numberOfElementsInQueue === 0) {
```

Comment:
Add a code comment to explain the requirement of if block^

Reply:
done